library/agent/AikidoDAST.ts

```diff
@@ -0,0 +1,7 @@
+import { Context } from "./Context";
+
+const AIKIDO_DAST_HEADER = "aikido-api-test";
+
+export function isAikidoDASTRequest(context: Context) {
+  return context.headers[AIKIDO_DAST_HEADER] === "1";
```

Comment:
Can `context.headers` be undefined here?

Reply:
Added some extra checks, even though the types say headers will always be an object